tests/prompts.test.ts

```diff
@@ -0,0 +1,398 @@
+      );
+      expect(prompt).toMatchObject({
+        categories: ["M/M", "Other"],
+      });
+    });
+
+      test("Fetches Prompt Categores (single)", async () => {
+        const prompt = await getPrompt(
+          await getPromptDetailsFromUrl({url: "https://www.archiveofourown.org/collections/mo_dao_zu_shi_kink_meme_2020/prompts/1927806"})
+        );
+        expect(prompt).toMatchObject({
+          categories: ["Gen"],
+        });  
+      });    
+
+    test("Fetches Prompt Categores (0 as null) with multiple fills", async () => {
+      const prompt = await getPrompt(
+        await getPromptDetailsFromUrl({url: "http://www.archiveofourown.org/collections/mo_dao_zu_shi_kink_meme_2020/prompts/1909048"})
+      );
+      expect(prompt).toMatchObject({
+        categories: null,
+      });
+    });      
+
+    test("Fetches collectionName & PromptID", async () => {
+      const prompt = await getPrompt(
+        await getPromptDetailsFromUrl({url: "http://www.archiveofourown.org/collections/mo_dao_zu_shi_kink_meme_2020/prompts/1909048"})
+      );
+      expect(prompt).toMatchObject({
+        collectionName: "mo_dao_zu_shi_kink_meme_2020",
+        id: "1909048"
+      });
+    }); 
+
+  });
+
+
+
+
+
+    
+
+
+    describe("Fetches full Prompt", () => {
+
+      test("Fetches multiple fills", async () => {
+        //https://archiveofourown.org/collections/mo_dao_zu_shi_kink_meme_2020/prompts/1909048
+        const prompt = await getPrompt({
+          promptId: "1909048",
+          collectionName: "mo_dao_zu_shi_kink_meme_2020"
+        });
+  
+        expect(prompt).toMatchObject({
+          postedAt: '30 Nov 2020',
+          summary: "<p>Jin Guangyao reincarnates as a cockroach in the Cloud Recesses. Lan Xichen is still in seclusion and enjoys having a little bug friend. Up to you if he knows or just thinks the roach is A-Yao because of a marking or something, or if he doesn't know at all and is just being a friend to all animals. And lonely.</p><p>Bonus: Da-ge spiderbro.</p><p>DNW: Bugfucking, the whole thing being humor. Plz make this stone-cold crack treated seriously.</p>",
+          collectionDisplayTitle: "Mó Dào Zǔ Shī | The Untamed Kink Meme 2020",
+          categories: null,
+          ratings: ["Not Rated"],
+          author: "Anonymous",
+          fandoms: ['陈情令 | The Untamed (TV)'],
+
+          tags: {
+            warnings: ["Creator Chose Not To Use Archive Warnings"],
+            characters: [],
+            relationships: ['Lan Huan | Lan Xichen/Meng Yao | Jin Guangyao'],
+            additional: ['Crack Treated Seriously'],
+          },
+          claims: {count: 3, isAnonCollection: true},
+          title: "jgy reincarnates as a cockroach",
+          collectionName: "mo_dao_zu_shi_kink_meme_2020",
+          id: "1909048",
+          filled: true,
+          // fills: [
```

Comment:
If these are commented out for a reason, write a comment about why, if not either uncomment or delete.